data/watermarks/simple-text.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg xmlns="http://www.w3.org/2000/svg"
+     x="0"
+     y="0">
+
+  <text x="0" y="5px" text-anchor="start"
+        fill="#$(WATERMARK_COLOR)"
+        font-family="$(WATERMARK_FONT_FAMILY)"
+        font-style="$(WATERMARK_FONT_STYLE)"
+        font-weight="$(WATERMARK_FONT_WEIGHT)"
+        font-size="10px">$(WATERMARK_TEXT)</text>
```

Comment:
Yes but this is the font-name. The family is retrieved specifically:

      g_snprintf(buffer, sizeof(buffer), "%s", pango_font_description_get_family(font));